app/src/main/java/com/example/yard/MainActivity.java

```diff
@@ -3,23 +3,33 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.content.Intent;
+import android.media.Image;
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
 
-    private Button btnGo;
+    private Button takePicture;
```

Comment:
Naming is important for code quality. Follow the name format recommended by CodePath. It's a button so it should be `btnTakePicture`.

Or usually I will use `takePictureButton`.

Similar to the naming of all the other fields here